bug: testing with Godot 4.3 crashes when game starts

## Scenario:

- Testing a Godot 4.3 game on the latest main branch
- Game built successfully
- Scanned QR code
- Logs shown but game crashes before starting

## ShipThis logs

```
11:51:15.922 [ShipThis] Downloading godot_v4_3 (22206714/66620144)
11:51:15.92  [FakeSplitInstallManager] availableSplits [, config.or, godot_v4_0.config.fa, config.pa, godot_v4_3.config.arm64_v8a, config.gu, 
11:51:16.922 [ShipThis] Downloading godot_v4_3 (44413428/66620144)
11:51:17.924 [ShipThis] Downloading godot_v4_3 (66620142/66620144)
11:51:18.937 [SplitCompat] Copying splits.
11:51:19.373 [SplitCompat] Splits copied.
11:51:19.437 [SplitCompat] addAssetPath completed with 14
11:51:19.438 [SplitCompat] addAssetPath completed with 13
11:51:19.438 [SplitCompat] Splits verified.
11:51:19.441 [SplitCompat] NativeLibraryExtractor: synchronizing native libraries
11:51:19.444 [SplitCompat] NativeLibraryExtractor: there are native libraries for supported ABI arm64-v8a; will use this ABI
11:51:19.444 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi
11:51:19.445 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi-v7a
11:51:19.446 [SplitCompat] NativeLibraryExtractor: using library libgodot_android_v4_3.so for ABI arm64-v8a
11:51:19.446 [SplitCompat] NativeLibraryExtractor: split 'godot_v4_3.config.arm64_v8a' has native library 'libgodot_android_v4_3.so' for ABI 'arm64-v8a'
11:51:19.454 [SplitCompat] NativeLibraryExtractor: split 'godot_v4_3.config.arm64_v8a' has native library 'libgodot_android_v4_3.so' that does not exist; 
11:51:19.696 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi-v7a
11:51:19.697 [SplitCompat] NativeLibraryExtractor: using library libgodot_android_v4_4.so for ABI arm64-v8a
11:51:19.699 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi
11:51:19.699 [SplitCompat] NativeLibraryExtractor: there are native libraries for supported ABI arm64-v8a; will use this ABI
11:51:19.701 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi-v7a
11:51:19.702 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI arm64-v8a
11:51:19.703 [SplitCompat] NativeLibraryExtractor: split 'godot_v4_4.config.arm64_v8a' has native library 'libgodot_android_v4_4.so' for ABI 'arm64-v8a'
11:51:19.704 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi
11:51:19.704 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi-v7a
11:51:19.704 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI armeabi
11:51:19.704 [SplitCompat] NativeLibraryExtractor: there are no native libraries for supported ABI arm64-v8a
11:51:19.708 [SplitCompat] addAssetPath completed with 23
11:51:19.709 [SplitCompat] addAssetPath completed with 21
11:51:19.710 [SplitCompat] addAssetPath completed with 24
11:51:19.711 [SplitCompat] addAssetPath completed with 22
11:51:19.711 [SplitCompat] Split 'godot_v4_3.config.arm64_v8a' installation emulated
11:51:19.711 [SplitCompat] Split 'godot_v4_4' installation emulated
11:51:19.711 [SplitCompat] Split 'godot_v4_3' installation emulated
11:51:19.714 [SplitCompat] Splits installed.
11:51:19.714 [SplitCompat] Split 'godot_v4_4.config.arm64_v8a' installation emulated
11:51:19.715 [ShipThis] godot_v4_3 installed, launching
```

## ADB logs

```
12-18 11:51:19.724  3124  3124 D AndroidRuntime: Shutting down VM
12-18 11:51:19.727  3124  3124 E AndroidRuntime: FATAL EXCEPTION: main
12-18 11:51:19.727  3124  3124 E AndroidRuntime: Process: com.shipthis.go, PID: 3124
12-18 11:51:19.727  3124  3124 E AndroidRuntime: java.lang.RuntimeException: Unable to instantiate activity ComponentInfo{com.shipthis.go/com.shipthis.go.GodotAppv4_3}
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4403)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4699)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:224)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:133)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:103)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:80)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2961)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:132)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.os.Looper.dispatchMessage(Looper.java:333)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:263)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:367)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:9287)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:566)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:929)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "com.shipthis.go.GodotAppv4_3" on path: DexPathList[[zip file "/data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/base.apk", zip file "/data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/split_config.arm64_v8a.apk", zip file "/data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/split_config.en.apk", zip file "/data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/split_config.xxhdpi.apk", zip file "/data/user/0/com.shipthis.go/files/splitcompat/1/verified-splits/godot_v4_4.apk", zip file "/data/user/0/com.shipthis.go/files/splitcompat/1/verified-splits/godot_v4_3.apk"],nativeLibraryDirectories=[/data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/lib/arm64, /data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/base.apk!/lib/arm64-v8a, /data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/split_config.arm64_v8a.apk!/lib/arm64-v8a, /data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/split_config.en.apk!/lib/arm64-v8a, /data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/split_config.xxhdpi.apk!/lib/arm64-v8a, /data/user/0/com.shipthis.go/files/splitcompat/1/native-libraries/godot_v4_4.config.arm64_v8a, /data/user/0/com.shipthis.go/files/splitcompat/1/native-libraries/godot_v4_3.config.arm64_v8a, /system/lib64, /system_ext/lib64]]
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:259)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:642)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:578)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.AppComponentFactory.instantiateActivity(AppComponentFactory.java:97)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at androidx.core.app.CoreComponentFactory.instantiateActivity(CoreComponentFactory.java:44)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.Instrumentation.newActivity(Instrumentation.java:1458)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4392)
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	... 14 more
12-18 11:51:19.727  3124  3124 E AndroidRuntime: 	Suppressed: java.io.IOException: Failed to open dex files from /data/app/~~KxwyU2DQgNGImxh6ntutVQ==/com.shipthis.go-cG1SldA2mQHjYKUwKfTZ0w==/split_config.arm64_v8a.apk because: Failed to find entry 'classes.dex': Entry not found
```

--- a/godot_v4_0/src/main/java/com/shipthis/go/GodotAppv4_0.java
+++ b/godot_v4_0/src/main/java/com/shipthis/go/GodotAppv4_0.java
@@ -28,7 +28,7 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-package com.godot.game;
+package com.shipthis.go;
 
 import org.godotengine.godotv4_0.FullScreenGodotApp;
 

--- a/godot_v4_1/src/main/java/com/shipthis/go/GodotAppv4_1.java
+++ b/godot_v4_1/src/main/java/com/shipthis/go/GodotAppv4_1.java
@@ -28,7 +28,7 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-package com.godot.game;
+package com.shipthis.go;
 
 import org.godotengine.godotv4_1.FullScreenGodotApp;
 

--- a/godot_v4_2/src/main/java/com/shipthis/go/GodotAppv4_2.java
+++ b/godot_v4_2/src/main/java/com/shipthis/go/GodotAppv4_2.java
@@ -28,7 +28,7 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-package com.godot.game;
+package com.shipthis.go;
 
 import org.godotengine.godotv4_2.GodotActivity;
 

--- a/godot_v4_3/src/main/java/com/shipthis/go/GodotAppv4_3.java
+++ b/godot_v4_3/src/main/java/com/shipthis/go/GodotAppv4_3.java
@@ -28,7 +28,7 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
-package com.godot.game;
+package com.shipthis.go;
 
 import org.godotengine.godotv4_3.GodotActivity;
 

